testscommon/realConfigsHandling.go

```diff
@@ -0,0 +1,105 @@
+package testscommon
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/common"
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateTestConfigs will try to copy the whole configs directory to a temp directory and return the configs after load
+// The copying of the configs is required because minor adjustments of their contents is required for the tests to pass
+func CreateTestConfigs(tb testing.TB, originalConfigsPath string) *config.Configs {
+	tempDir := tb.TempDir()
+
+	newConfigsPath := path.Join(tempDir, "config")
+
+	// TODO refactor this cp to work on all OSes
+	cmd := exec.Command("cp", "-r", originalConfigsPath, newConfigsPath)
```

Comment:
why do you need this copy? since you don't change the files, you could have used them as they are